File: M8/Exemplos/App_eXperience_Java/app/src/main/java/com/elgin/elginexperience/shipay/models/OrderItem.java
package com.elgin.elginexperience.shipay.models;

import com.google.gson.annotations.SerializedName;

public class OrderItem {
    public OrderItem(String item_title, float unit_price, int quantity) {
        this.item_title = item_title;
        this.unit_price = unit_price;
        this.quantity = quantity;
    }

    @SerializedName("item_title")
    public String item_title;
    @SerializedName("unit_price")
    public float unit_price;
    @SerializedName("quantity")
    public int quantity;
}

File: Exemplos/App_eXperience_Kotlin/app/src/main/java/com/example/e1_elgin_kotlin/shipay/models/OrderItem.kt
package com.example.e1_elgin_kotlin.shipay.models

import com.google.gson.annotations.SerializedName

data class OrderItem(
    @field:SerializedName("item_title") var item_title: String,
    @field:SerializedName("unit_price") var unit_price: Float,
    @field:SerializedName("quantity") var quantity: Int
)
